src/soilbiogeochem/SoilBiogeochemDecompCascadeConType.F90

```diff
@@ -99,6 +133,107 @@ subroutine init_decomp_cascade_constants()
     decomp_cascade_con%is_lignin(0:ndecomp_pools)                      = .false.
     decomp_cascade_con%spinup_factor(1:ndecomp_pools)                  = nan
 
+    decomp_cascade_con%Ntrans_setup = (ndecomp_cascade_transitions-ndecomp_cascade_outtransitions)*nlevdecomp
+    decomp_cascade_con%Ntri_setup   = (3*nlevdecomp-2)*(ndecomp_pools - 1) !exclude one cwd 
   end subroutine init_decomp_cascade_constants
 
+ subroutine InitSoilTransfer()
+
```

Comment:
Add some comments to define what this subroutine does

Reply:
done